mcstatus/tests/test_timeout.py

```diff
@@ -29,8 +29,7 @@ def test_tcp_socket_read(self):
             from asyncio import TimeoutError
 
         loop = asyncio.get_event_loop()
-        with patch("asyncio.open_connection") as open_conn:
-            open_conn.return_value = (FakeAsyncStream(), None)
+        with patch("asyncio.open_connection", fake_asyncio_asyncio_open_connection):
```

Comment:
Holidays slowed me down a bit, but I'll get this change in there. I had the same changes locally before Christmas.